EIPS/eip-4955.md

```diff
@@ -0,0 +1,197 @@
+---
+eip: 4955
+title: Vendor Specific Metadata Extension for Non-Fungible Tokens
+description:  Add a new field to NFT metadata to store vendor specific data
+author: Ignacio Mazzara (@nachomazzara)
+discussions-to: https://ethereum-magicians.org/t/eip-4955-non-fungible-token-metadata-namespaces-extension/8746
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-03-29
+requires: 721, 1155
```

Comment:
Could be the space here, but I'm not sure. I'll poke lightclient about it.

Reply:
@SamWilsn I have removed some white spaces. Not sure if that will help.